docs/rfcs/RFC0002-MonoRepo-Gardener-Rotations.md

```diff
@@ -0,0 +1,137 @@
+# Mono-repo Gardener
+
+Gardeners<sup>[1]</sup> are responsible for keeping their mono-repos shippable;
+ensuring bad commits get reverted by keeping build, test, and nightly release
+workflows green so that developers can remain productive and bugs are caught
+closer to commit times.
```

Comment:
Phrasing as "bad commits" may be overly negative, especially if we don't define our terms carefully. I think this also inverts cause and effect: tree health is a factor of having automated workflows continually passing ("green"), and reverting "bad commits" quickly is one tool to achieve that. 

How about rewording to something along these lines instead?

> Gardeners are responsible for keeping their component areas continually shippable. Build, test, and nightly release workflows should be kept passing ("green"). When tests fail ("red") or builds fail ("black"), corrective action like identifying and reverting culprit commits should be taken quickly, so that developers can remain productive and bugs are caught closer to commit times.